packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1452,6 +1451,11 @@ export const createAzureCommunicationCallAdapter = async ({
   /* @conditional-compile-remove(PSTN-calls) */ alternateCallerId,
   /* @conditional-compile-remove(video-background-effects) */ options
 }: AzureCommunicationCallAdapterArgs): Promise<CallAdapter> => {
+  if (isMicrosoftTeamsUserIdentifier(userId)) {
+    throw new Error(
+      'Microsoft Teams user identifier is not supported by AzureCommunicationCallAdapter, please use our TeamsCallAdapter.'
```

Comment:
nitty nit: avoid possessive adjective
```suggestion
      'Microsoft Teams user identifier is not supported by AzureCommunicationCallAdapter. Instead use TeamsCallAdapter.'
```